Examples/Swift/SettingsItems.swift

```diff
@@ -0,0 +1,65 @@
+import UIKit
+import MapboxDirections
+
+
+extension SettingsViewController {
+    
+    func sections() -> [Section] {
+        
+        let offlineItem = Item(title: "Download arbitrary region", viewControllerType: OfflineViewController.self, payload: nil)
+        let downloadSFItem = Item(title: "Download SF region", viewControllerType: nil, payload: {
+            
+            Directions.shared.availableOfflineVersions(completionHandler: { (versions, error) in
+                guard let version = versions?.first else { return }
+                let sfBoundingBox = BoundingBox([CLLocationCoordinate2D(latitude: 37.7890, longitude: -122.4337),
+                                                 CLLocationCoordinate2D(latitude: 37.7881, longitude: -122.4318)])
+                
+                Directions.shared.downloadTiles(for: sfBoundingBox, version: version, completionHandler: { (url, response, error) in
+                    guard let url = url else { return }
+                    print("Downloaded \(url)")
```

Comment:
Shouldn't we unpack at this point?

Reply:
Ended up removing this example since it's basically the same as downloading an arbitrary region.